open-idea-station/Acorn
# Add coming soon to master branch

#### What does this PR do?
Add a coming soon page to OIS

#### Description of Task proposed in this pull request?
- Add a new page that displays `coming soon`

#### How should this be manually tested?
- Pull the branch
- Start your server
- Visit any route to see the page

#### What are the relevant pivotal tracker stories?
- [164520767](https://www.pivotaltracker.com/story/show/164520767)

#### Any background context you want to add?
None

#### What I have learned working on this feature: [If you don't put anything here, you are doing it wrong!]
- How to route all request to a controller

#### Screenshots: [If you made some visual changes to the application please upload screenshots here, or remove this section]
<img width="1440" alt="Screen Shot 2019-03-11 at 1 41 03 PM" src="https://user-images.githubusercontent.com/32465536/54126175-3d92c700-4407-11e9-9c30-350be4c79a19.png">
